sparkcode-v1.py

```diff
@@ -21,12 +21,12 @@ async def on_ready():
 
 @client.command()
 async def ping(ctx):
-"""Test the bot's skills on ping-pong."""
+    """Test the bot's skills on ping-pong."""
```

Comment:
Didn't even know that those were causing issues

Reply:
They're causing issues in Python 3.8.